indexer/services/comlink/__tests__/controllers/api/v4/asset-positions-controller.test.ts

```diff
@@ -148,18 +150,164 @@ describe('asset-positions-controller#V4', () => {
       const response: request.Response = await sendRequest({
         type: RequestMethod.GET,
         path: `/v4/assetPositions?address=${testConstants.defaultAddress}` +
-          `&subaccountNumber=${testConstants.defaultSubaccount.subaccountNumber}`,
+            `&subaccountNumber=${testConstants.defaultSubaccount.subaccountNumber}`,
```

Comment:
nit: why'd you change the indentation on all of these lines above?

Reply:
not intentional. ide seems to have a mind of of its own
You can look at the changes without whitespace and it should make it easier to review